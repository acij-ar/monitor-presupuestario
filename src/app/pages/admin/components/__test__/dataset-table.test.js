const React = require('react');
const {shallow} = require('enzyme');

const DatasetTable = require('../dataset-table');

describe('Dataset table', () => {
  it('should match snapshot', () => {
    const props = {
      datasets: [
        {
          filename: '2007.csv',
          upToDate: true,
          lastModified: '2020-03-02T19:16:34.003Z',
        },
        {
          filename: '2008.csv',
          upToDate: false,
          lastModified: null,
        },
      ]
    };

    const wrapper = shallow(<DatasetTable {...props} />);
    expect(wrapper).toMatchSnapshot();

  });
});
