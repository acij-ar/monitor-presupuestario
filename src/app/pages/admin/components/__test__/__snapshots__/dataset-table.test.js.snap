// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dataset table should match snapshot 1`] = `
<table>
  <thead>
    <tr>
      <th>
        Dataset
      </th>
      <th>
        Número de filas
      </th>
      <th>
        Última actualización
      </th>
      <th>
        Al dia con google drive
      </th>
      <th />
    </tr>
  </thead>
  <tbody>
    <tr
      key="2007.csv"
    >
      <td>
        2007.csv
      </td>
      <td>
        2020-03-02T19:16:34.003Z
      </td>
      <td>
        ✅
      </td>
    </tr>
    <tr
      key="2008.csv"
    >
      <td>
        2008.csv
      </td>
      <td>
        -
      </td>
      <td>
        ❌
      </td>
    </tr>
  </tbody>
</table>
`;
